tests/unit/source_code/test_jobs.py

```diff
@@ -88,19 +88,6 @@ def test_workflow_task_container_builds_dependency_graph_empty_task(mock_path_lo
     ws.assert_not_called()
 
 
-def test_workflow_task_container_builds_dependency_graph_pytest_pypi_library(mock_path_lookup, graph) -> None:
-    ws = create_autospec(WorkspaceClient)
-    libraries = [compute.Library(pypi=compute.PythonPyPiLibrary(package="demo-egg"))]  # installs pkgdir
```

Comment:
This test explicitly tests for a pypi install, which is covered in the integration tests also